README.md

```diff
@@ -2,6 +2,15 @@
 
 This repository contains implementation of the protocol to exchange ERC20 compliant tokens using fully decentralized, on-chain order book and matching engine. The protocol is used by OasisDEX, eth2dai.com and many other DeFi projects. 
 
+
+## Development
+
+```
+dapp build
```

Comment:
```suggestion
dapp --use solc:0.5.12 build
```

Reply:
Actually we should remove this line as the script themselves have the building part.

Reply:
Done in my PR